
segment_time: 2.0
frequency: 200
kp: 10.0
kd: 1.0
torque: 0.0
pos_gain: 1.0
vel_gain: 1.0
ff_gain: 1.0
v_max: 5.0
loop: true
loop_count: 2
record_all_loops: false
control_mode: "velocity"
